lib/globalize/active_record/act_macro.rb

```diff
@@ -42,7 +35,6 @@ def allow_translation_of_attributes(attr_names)
 
       def apply_globalize_options(options)
         options[:table_name] ||= "#{table_name.singularize}_translations"
-        options[:foreign_key] ||= class_name.foreign_key
```

Comment:
Shouldn't this still be able to be specified in the case that we can't find it via reflection?